hal/Cargo.toml

```diff
@@ -16,7 +16,7 @@ embedded-hal = "~0.2"
 nb = "~0.1"
 paste = "~0.1"
 bitfield = "~0.13"
-vcell = "~0.1"
+vcell = { version = "~0.1", features = ["const-fn"], git = "https://github.com/japaric/vcell.git" }
```

Comment:
git deps aren't allowed on crates.io

Reply:
Yeah, I was originally using the git version because `const-fn` support hasn't been published yet, however that was before I was able to leverage the DMAC registers defined in the PAC, so this should be completely unnecessary. I just have to wire everything up (at which point I can remove this)